src/state/runner/sagas/PollRunnerState.js

```diff
@@ -97,7 +101,7 @@ function* startPolling(action) {
   stopActionName += '_' + action.runnerId;
 
   // Prevent double polling on same scenario
-  yield put(forgeStopPollingAction(action.runnerId));
+  // yield put(forgeStopPollingAction(action.runnerId));
```

Comment:
If this line is no longer required, then we should probably remove it :smile:

Reply:
That's strange, I tried to delete this action and it broke polling, I will retry it one more time to see why it's not broken this time :sweat_smile:

Reply:
Yes, we still need it to avoid a loop of update in Redux, I'll fix it, thanks ! :pray:

Reply:
UPD: we still need this action on line 92 when the run stops but not on this line where it prevents the polling on scenario change but isn't necessary for runners as we don't fetch the runner by id on scenario change